app/src/main/java/chat/rocket/android/authentication/server/ui/ServerFragment.kt

```diff
@@ -188,47 +216,49 @@ class ServerFragment : Fragment(), ServerView {
     }
 
     override fun updateServerUrl(url: HttpUrl) {
-        if (activity != null && view != null) {
-            if (url.scheme() == "https") text_server_protocol.setSelection(0) else text_server_protocol.setSelection(1)
-            protocol = "${url.scheme()}://"
+        ui {
+            if (url.scheme() == "https") {
+                spinner_server_protocol.setSelection(0)
+            } else {
+                spinner_server_protocol.setSelection(1)
+            }
 
-            val serverUrl = url.toString().removePrefix("${url.scheme()}://")
-            text_server_url.textContent = serverUrl
+            protocol = "${url.scheme()}://"
+            text_server_url.textContent = url.toString().removePrefix("${url.scheme()}://")
```

Comment:
I think you meant to put `text_server_url.textContent = url.toString().removePrefix(protocol)` :thinking:

Reply:
Good catch!!
👍